openapi/trello/client.bal

```diff
@@ -1034,23 +1024,21 @@ public isolated client class Client {
     }
     # Get boards by ID board by field
     #
-    # + idBoard - Board_id
-    # + 'field - Field
-    # + return - Success
+    # + idBoard - Board_id 
+    # + 'field - Field 
+    # + return - Success 
     remote isolated function getBoardsByIdBoardByField(string idBoard, string 'field) returns http:Response|error {
         string  path = string `/boards/${idBoard}/${'field}`;
-        map<anydata> queryParam = {'key: self.apiKeys["key"], token: self.apiKeys["token"]};
-        path = path + check getPathForQueryParam(queryParam);
         http:Response response = check self.clientEp-> get(path, targetType = http:Response);
```

Comment:
API keys `key` and `token` are missing in this operation. Please confirm whether they are not needed or not added mistakenly.